ocs_ci/ocs/resources/rgw.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, namespace=None):
             kind="storageclass", namespace=namespace, resource_name=sc_name
         )
         self.s3_internal_endpoint = (
```

Comment:
A clarification: this class member was not used until now in OCS-CI, and this old way of getting the internal endpoint is no longer working.